src/duct/middleware/web.clj

```diff
@@ -12,22 +12,22 @@
 (def ^:private request-log-keys
   [:request-method :uri :query-string])
 
-(defn- log-request [logger request]
-  (logger/log logger :info ::request (select-keys request request-log-keys)))
+(defn- log-request [logger request level]
+  (logger/log logger level ::request (select-keys request request-log-keys)))
 
 (defn- log-error [logger ex]
   (logger/log logger :error ::handler-error ex))
 
 (defn wrap-log-requests
   "Log each request using the supplied logger. The logger must implement the
   duct.core.protocols/Logger protocol."
-  [handler logger]
+  [handler logger {:keys [level] :or {level :info}}]
```

Comment:
Can you add in a 2-arity version with a blank map? i.e.

```clojure
(defn wrap-log-requests
  "..."
  ([handler logger]
    (wrap-log-requests handler logger {}))
  ([handler logger {:keys [level] :or {level :info}}]
    ...))
```

Reply:
Sure. I will do it.